tests/flow/test_list.py

```diff
@@ -180,23 +180,23 @@ def test04_head_function(self):
             actual_result = redis_graph.query(query)
             self.env.assertTrue(False)
         except ResponseError as e:
-            self.env.assertContains("Type mismatch: expected List but was Integer", str(e))
+            self.env.assertContains("Type mismatch: expected List, or Null but was Integer", str(e))
```

Comment:
last ','  need to be omitted

Reply:
If there are two items, the last comma should be omitted.
If there are more than two, the last comma should be present (US grammar conversion)
https://www.grammar-monster.com/lessons/commas_in_lists.htm

Reply:
Ok, change done to follow the US grammar convention. [f55591f](https://github.com/RedisGraph/RedisGraph/pull/2691/commits/f55591f8e396620eb2d18eec6a0c2be676f92b87)